server/data/personClient.ts

```diff
@@ -25,9 +25,10 @@ export default class PersonClient {
     this.restClient = new RestClient('personClient', config.apis.approvedPremises as ApiConfig, token)
   }
 
-  async search(crn: string): Promise<Person> {
+  async search(crn: string, checkCaseload: boolean): Promise<Person> {
+    const path = `${paths.people.search({})}?crn=${crn}${checkCaseload ? '&checkCaseload=1' : ''}`
```

Comment:
Why use `1` over `true` as the value for `checkCaseload`? 
I wonder if we could use `qs` to make this slightly neater?

Reply:
Actually, scratch that. I'll go with `true` as I've tested it locally and it works fine 👍